server/gateway/src/utils.ts

```diff
@@ -1,11 +1,15 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
Let's not check this in. I would rather have logic to explicitly assert or throw then return an unsafe type.